packages/markup-frontstage/package.json

```diff
@@ -0,0 +1,116 @@
+{
+  "name": "@bentley/markup-frontstage-react",
+  "version": "1.0.0",
+  "description": "Markup Frontstage React",
+  "license": "MIT",
+  "main": "lib/markup-frontstage-react.js",
+  "typings": "lib/markup-frontstage-react",
+  "scripts": {
+    "copy:assets": "cpx \"./src/**/*.scss\" ./lib && cpx \"./public/**/*\" ./lib/public",
+    "pseudolocalize": "betools pseudolocalize --englishDir ./public/locales/en --out ./lib/public/locales/en-PSEUDO",
+    "build": " tsc 1>&2 && npm run copy:assets && npm run pseudolocalize && npm run copy-css",
+    "build-prod": " tsc -p tsconfig.json 1>&2 && npm run copy:assets && npm run copy-css",
+    "clean": "rimraf lib build",
+    "lint": "tslint -p . 1>&2",
+    "test": "jest",
+    "copy-css": "copyfiles -u1 \"./src/**/*.scss\" ./lib",
+    "cover": "npm test"
+  },
+  "keywords": [
+    "Bentley",
+    "iModel Markup Frontstage React",
+    "iModel.js"
+  ],
+  "author": {
+    "name": "Bentley Systems, Inc.",
+    "url": "http://www.bentley.com"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/imodeljs/viewer-components-react/packages/markup-frontstage"
+  },
+  "dependencies": {
+    "classnames": "^2.2.6",
+    "i18next": "^10.2.2",
+    "i18next-browser-languagedetector": "^4.0.2"
+  },
+  "devDependencies": {
+    "@babel/plugin-proposal-class-properties": "^7.8.3",
+    "@babel/plugin-transform-runtime": "^7.9.6",
+    "@babel/preset-env": "^7.9.6",
+    "@babel/preset-react": "^7.9.4",
+    "@babel/preset-typescript": "7.9.0",
+    "@bentley/bentleyjs-core": "^2.0.0",
+    "@bentley/build-tools": "^2.0.0",
+    "@bentley/imodeljs-common": "^2.0.0",
+    "@bentley/imodeljs-frontend": "^2.0.0",
+    "@bentley/imodeljs-i18n": "^2.0.0",
+    "@bentley/imodeljs-markup": "^2.0.0",
+    "@bentley/presentation-frontend": "^2.0.0",
+    "@bentley/ui-abstract": "^2.0.0",
+    "@bentley/ui-components": "^2.0.0",
+    "@bentley/ui-core": "^2.0.0",
+    "@bentley/ui-framework": "^2.0.0",
```

Comment:
core bentley imjs deps shouldnt be using `^` and should be pinned

Reply:
I've been meaning to ask about this, any particular reason we shouldn't use them with the `^` in these types of packages?  We don't do it in the iModel.js repository because of the linking directory within the repo but when the packages aren't built within the same repo I don't see an issue with using the `^` to match the peerDep semver.

Reply:
I think now that we're off of dev builds we should use the ^ and expect versioning is done correctly on the core side. This also provides pressure for that to be the case. The bigger concern would be if there was a backend component this relied on that didn't update at the same time, since it could be problematic for a newer frontend to connect to an older backend, but that isn't applicable in this case

Reply:
sounds good to me